.openpublishing.publish.config.json

```
@@ -7,7 +7,7 @@
       "locale": "en-us",
       "monikers": [],
       "moniker_ranges": [
-        ">= webview2-winrt-1.0.992.28 <= webview2-winrt-1.0.2849.39 || >= webview2-winrt-1.0.955-prerelease <= webview2-winrt-1.0.2895-prerelease"
+        ">= webview2-winrt-1.0.992.28 <= webview2-winrt-1.0.2849.39 || >= webview2-winrt-1.0.955-prerelease <= webview2-winrt-1.0.2895-prerelease || webview2-winrt-preview"
```

```suggestion
        ">= webview2-winrt-1.0.992.28 <= webview2-winrt-1.0.2903.40 || >= webview2-winrt-1.0.955-prerelease <= webview2-winrt-1.0.2895-prerelease"
```

We can remove the temp moniker `webview2-winrt-preview` now.